lab/lab1/part4-project-euler.html

```diff
@@ -32,7 +32,47 @@
 
     ===================== */
 
-
+    /*problem 1
+    If we list all the natural numbers below 10 that are multiples of 3 or 5, we
+    get 3, 5, 6 and 9. The sum of these multiples is 23.
+    Find the sum of all the multiples of 3 or 5 below 1000.*/
+
+    var array = [];
+    let sum = 0;
+    for(var i=0; i<1000; i++) {
+      if (i%3 === 0 || i%5 === 0) {
+        array.push(i);
+        sum += i;
+      }
+    };
+
+    console.log(sum);
+
+    /*problem 2
+    Each new term in the Fibonacci sequence is generated by adding the previous
+    two terms. By starting with 1 and 2, the first 10 terms will be:
+    1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...
+    By considering the terms in the Fibonacci sequence whose values do not
+    exceed four million, find the sum of the even-valued terms.*/
+
+    let evenSum = 0;
+    let fibArray = [1,2];
+    var x=0;
+    var y=0;
+
+    while (fibArray[0] < 4000000) {
+      for (var j=0; j<1; j++) {
+        if (fibArray[j]%2 === 0) {
+          evenSum += fibArray[j];
+        }
+      }
+      x=fibArray[0];
+      y=fibArray[1];
+      fibArray[0]=x+y;
+      fibArray[1]=fibArray[0]+y;
+    };
```

Comment:
the correct answer is: 4613732. You got: 881144
[according to](https://stackoverflow.com/questions/9053545/finding-the-sum-of-even-valued-terms-in-fibonacci-sequence)